ui/Comments/MainComment.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { Card } from "react-native-elements";
+import { StyleSheet, View, Text } from "react-native";
+import Title from "./Title";
+
+import moment from "moment";
+
+type CommentProps = {
+  comment: IComment;
+  onReply: () => void;
+};
+
+const commentCreationTime = (created_at: string) => {
+  let creationTime = moment(created_at);
+  let currentTime = moment();
+
+  let msg: string | null = null;
+
+  let timeIntervals = [
+    "seconds",
+    "minutes",
+    "hours",
+    "months",
+    "days",
+    "years",
+  ];
+  let msgs = ["now", "min", "hr", "month", "day", "ys"];
+  let index = 0;
+  for (; index < timeIntervals.length - 1; index++) {
+    let dif = currentTime.diff(creationTime, timeIntervals[index + 1]);
+    if (dif === 0) break;
+  }
+  index--;
+
+  if (index === 0) {
+    msg = msgs[0];
+  } else {
+    let dif = currentTime.diff(creationTime, timeIntervals[index]);
+    if (dif === 1) {
+      msg = dif.toString() + " " + msgs[index] + " ago";
+    } else {
+      msg = dif.toString() + " " + msgs[index] + "s ago";
+    }
+  }
+
+  return msg;
+};
```

Comment:
You might want to see this :) [Moment: Time from now](https://momentjs.com/docs/#/displaying/fromnow/)

Reply:
lol